### PR TITLE
Updatating API documentation - Adding examples for Microsoft.Quantum.Arrays.Any

### PR DESCRIPTION
**Pull request by akshob**
Adding an example snippet for Microsoft.Quantum.Arrays.Any documentation.

- quantum-docs-pr: [https://github.com/MicrosoftDocs/quantum-docs-pr/pull/708](https://github.com/MicrosoftDocs/quantum-docs-pr/pull/708)
- Build successful
- [Preview URL](https://review.docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.arrays.any?view=qsharp-preview&branch=pr-en-us-708)

### CODE REVIEW

File: Standard/src/Arrays/Any.qs [hunk 1]
```diff
@@ -29,6 +29,19 @@ namespace Microsoft.Quantum.Arrays {
     ///
     /// # Output
     /// A `Bool` value of the OR function of the predicate applied to all elements.
+    ///
+    /// # Example
+    /// ```qsharp
+    ///open Microsoft.Quantum.Intrinsic;
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]:
```suggestion
    /// open Microsoft.Quantum.Intrinsic;
```

File: Standard/src/Arrays/Any.qs [hunk 2]
```diff
@@ -29,6 +29,19 @@ namespace Microsoft.Quantum.Arrays {
...
+    ///open Microsoft.Quantum.Arrays;
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]:
```suggestion
    /// open Microsoft.Quantum.Arrays;
```

File: Standard/src/Arrays/Any.qs [hunk 3]
```diff
@@ -29,6 +29,19 @@ namespace Microsoft.Quantum.Arrays {
...
+    ///open Microsoft.Quantum.Logical;
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]:
```suggestion
    /// open Microsoft.Quantum.Logical;
```

File: Standard/src/Arrays/Any.qs [hunk 4]
```diff
@@ -29,6 +29,19 @@ namespace Microsoft.Quantum.Arrays {
...
+    ///
+    ///operation ArrayAny_Demo() : Unit {
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]: I had a couple suggestions here relating to the [Q# style guide](https://docs.microsoft.com/quantum/contributing/style-guide#operations-and-functions) and the [draft API principles](https://github.com/MicrosoftDocs/quantum-docs-pr/pull/691):

- Underscores aren't suggested for use in operation and function names.
- This callable uses only classical and deterministic logic, and thus could be written as a function instead.

```suggestion
    /// operation IsThreePresent() : Bool {
```

**Comment by akshob** [comment]
Hi @cgranade 
Thank you for the suggestions and helping me get through my very first open source PR!
Had one question
- In the draft API principles, I see there is Acrolinx Scorecards. How do I get that?

And I will change the operation to function and send out an iteration soon!

**Comment by cgranade** [comment]
Happy to help, @akshob! With respect to your question, we don't currently support Acrolinx scorecards for Q# libraries, those only appear on the documentation repo (i.e.: where the draft API principles are hosted) at the moment.

**Comment by akshob** [comment]
Hi @cgranade 

This might be a very silly question. But how do I call a function defined in .qs from .py?
So far I have `Operation.qs`
```
namespace ArrayDemo {
    open Microsoft.Quantum.Intrinsic;
    open Microsoft.Quantum.Arrays;
    open Microsoft.Quantum.Logical;

    operation ArrayDemo() : Unit {
        let arrayOfInts = [1, 2, 3, 4, 5];
        let is3Present = IsNumberPresentInArray(3, arrayOfInts);
        Message($"{is3Present}");
    }

    function IsNumberPresentInArray(n: Int, array: Int[]): Bool {
        return Any(EqualI(_, n), array);
    }
}
```
And `array_any.py`. This works totally fine.
```
import qsharp
from ArrayDemo import ArrayDemo

ArrayDemo.simulate()
```

I tried the following 
```
import qsharp
from ArrayDemo import IsNumberPresentInArray

print(IsNumberPresentInArray(3, [1,2,3]))
```

but I get this error
```
(qsharp) nitrogen:quantum-development akgop$ python array_any.py
Preparing Q# environment...
Traceback (most recent call last):
  File "array_any.py", line 7, in <module>
    IsNumberPresentInArray(3, [1,2,3]).simulate()
TypeError: 'IsNumberPresentInArray' object is not callable
```

**Comment by akshob** [comment]
The Acrolinx scorecards don't appear in my pull request in documentation repo as well: https://github.com/MicrosoftDocs/quantum-docs-pr/pull/708

**Comment by cgranade** [comment]
> This might be a very silly question. But how do I call a function defined in `.qs` from `.py`?

Not silly at all, happy to help! For what it's worth, the Python interface is largely defined in https://github.com/microsoft/iqsharp, so that can also be a good place to look.

> ```
> import qsharp
> from ArrayDemo import IsNumberPresentInArray
> 
> print(IsNumberPresentInArray(3, [1,2,3]))
> ```

Note that `IsNumberPresentInArray` isn't a Python callable, but an object representing the Q# callable within Python that exposes several different methods to help you choose the best target machine for your needs. For instance, the `simulate` method will let you run `IsNumberPresentInArray` on the quantum simulator provided with the Quantum Development Kit. The main difference with the C# interface is that you'll need to provide the names of each input as well:

```
import qsharp
from ArrayDemo import IsNumberPresentInArray

print(IsNumberPresentInArray.simulate(n=3, array=[1,2,3]))
```

**Comment by akshob** [comment]
[Preview URL] (https://review.docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.arrays.any?view=qsharp-preview&branch=pr-en-us-708)

File: Standard/src/Arrays/Any.qs [hunk 5]
```diff
@@ -29,6 +29,22 @@ namespace Microsoft.Quantum.Arrays {
     ///
     /// # Output
     /// A `Bool` value of the OR function of the predicate applied to all elements.
+    ///
+    /// # Example
+    /// ```qsharp
+    /// open Microsoft.Quantum.Arrays;
+    /// open Microsoft.Quantum.Logical;
+    ///
+    /// function IsThreePresent(): Bool {
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]:
```suggestion
    /// function IsThreePresent() : Bool {
```

File: Standard/src/Arrays/Any.qs [hunk 6]
```diff
@@ -29,6 +29,22 @@ namespace Microsoft.Quantum.Arrays {
...
+    ///     let arrayOfInts = [1, 2, 3, 4, 5];
+    ///     let is3Present = IsNumberPresentInArray(3, arrayOfInts);
+    ///     return is3Present;
+    /// }
+    ///
+    /// function IsNumberPresentInArray(n: Int, array: Int[]): Bool {
```
Comment (cgranade) [inline Standard/src/Arrays/Any.qs]:
```suggestion
    /// function IsNumberPresentInArray(n : Int, array : Int[]) : Bool {
```

File: Standard/src/Arrays/Any.qs (hunk 1 above)
Comment (cgranade) [reply Standard/src/Arrays/Any.qs]: Marking as resolved, since this was addressed in 1005e64.

File: Standard/src/Arrays/Any.qs (hunk 3 above)
Comment (cgranade) [reply Standard/src/Arrays/Any.qs]: Marking as resolved, since this was addressed in 1005e64.

File: Standard/src/Arrays/Any.qs (hunk 4 above)
Comment (cgranade) [reply Standard/src/Arrays/Any.qs]: Marking as resolved, since this was addressed in 1005e64.

**Comment by cgranade** [comment]
Thank you for your contribution, @akshob!

**Pull request merged**